registration/migrations/0006_auto_20190526_2159.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
This just reminds me that I forgot to commit a migration on the visit thing.

Reply:
Bahaha classic